### ISSUE TITLE
Configure OVN access in LXD

### ISSUE THREAD

**Issue by stgraber** [canonical/microcloud#64]
When MicroOVN is set up, we should have MicroCloud do:
 - Pull list of services from MicroOVN API
 - Identify what servers run the `central` service
 - From that compute the northbound connection string and set it as `network.ovn.northbound_connection`
 - Whenever a server is added or removed, go through that process again

The connection string looks something like: `tcp:[2602:fc62:a:101::100]:6641,tcp:[2602:fc62:a:101::101]:6641,tcp:[2602:fc62:a:101::102]:6641`

So basically comma-separated list of `tcp:<IP>:6641` for each of the central servers.
